misc/acrn-config/board_config/new_board_kconfig.py

```diff
@@ -28,7 +28,7 @@
 }
 
 
-def find_avl_memory(ram_range, hpa_size):
+def find_avl_memory(ram_range, hpa_size, hv_start_offset):
```

Comment:
need to add description for new added option

Reply:
Done!